packages/oslo-generator-shacl-template/test/CardinalityConstraintHandler.unit.test.ts

```diff
@@ -0,0 +1,171 @@
+/**
+ * @group unit
+ */
+import 'reflect-metadata';
+import type { Logger } from '@oslo-flanders/core';
```

Comment:
<img width="885" alt="Screenshot 2024-04-11 at 15 10 47" src="https://github.com/Informatievlaanderen/OSLO-UML-Transformer/assets/28532087/1fd0fff1-5cb6-4ebe-9fba-f7198a28b21c">
I had some import issues with a lot of the test-cases. Do you have these as well?